src/NServiceBus.Testing/Invocations.cs

```diff
@@ -16,12 +16,14 @@ abstract class ExpectedInvocation<T> : IExpectedInvocation where T : ActualInvoc
         public void Validate(params ActualInvocation[] invocations)
         {
             var calls = invocations.Where(i => typeof(T) == i.GetType());
-            var success = calls.Any(c =>
-                                         {
-                                             var result = Validate(c as T);
-                                             
-                                             return result;
-                                         });
+
+            var callResults = calls.Select(c =>
+            {
+                var result = Validate(c as T);
+                return result;
```

Comment:
I'm open to suggestions :)

It needs to evaluate the `Validate` method for ALL instances of the invocations, and then give a bool result if one of the calls resulted in a true.

A simple `.Any()` stops processing on the first one that returns true, so I'm not sure if there's a better way to do the projection on all items and then following up with an Any()?